core/src/main/java/com/linecorp/armeria/internal/metric/RequestMetricSupport.java

```diff
@@ -200,4 +236,43 @@ public Timer totalDuration() {
             return totalDuration;
         }
     }
+
+    private static class DefaultClientRequestMetrics
+            extends AbstractRequestMetrics implements ClientRequestMetrics {
+
+        private final Counter writeTimeout;
+        private final Counter responseTimeout;
+
+        DefaultClientRequestMetrics(MeterRegistry parent, MeterIdPrefix idPrefix) {
+            super(parent, idPrefix);
+            writeTimeout = parent.counter(idPrefix.name("writeTimeout"), idPrefix.tags());
+            responseTimeout = parent.counter(idPrefix.name("responseTimeout"), idPrefix.tags());
+        }
+
+        @Override
+        public Counter writeTimeout() {
+            return writeTimeout;
+        }
+
+        @Override
+        public Counter responseTimeout() {
+            return responseTimeout;
+        }
+    }
+
+    private static class DefaultServiceRequestMetrics
+            extends AbstractRequestMetrics implements ServiceRequestMetrics {
+
+        private final Counter requestTimeout;
+
+        DefaultServiceRequestMetrics(MeterRegistry parent, MeterIdPrefix idPrefix) {
+            super(parent, idPrefix);
+            requestTimeout = parent.counter(idPrefix.name("requestTimeout"), idPrefix.tags());
```

Comment:
- `requestTimeouts`
- Or should we name it `timeouts` and add a tag? e.g. `timeouts{cause=ResponseTimeoutException}`